### DISCUSSION TITLE
can we have a guide how to use this?

### DISCUSSION THREAD

**Discussion by jacksonm36** [stefanwerfling/flyingfish#33] [General]
Hey your installation document is great and all despite a typo in the docker-compose.yaml. but i get stuck after installing it i have no idea how to setup a proxy give it ssl cert  etc. i tried google-ing it but  nothing came up.  does it even come with wildcard support?

**Comment by stefanwerfling** [comment]
Hello jacksonm36, the documentation is not where I would like it to be. Other projects are currently eating up a lot of my time.

First the question, did you find a typo?

The setup process would be as follows:
1.Installation
2. Port Releases
3. Create domains
4. Routes on the Nginx + SSL
5. (Optional, DynDNS client)
6. (Optional blacklist/whitelist)

If the 1st and 2nd have already done. Start with 3. and create your domain under Domains. They don't have to have a record if they don't want to operate the DNS server themselves.

As soon as the domains are created, you can set the routes and select the domain there.

I'll see if I can expand the documentation with these steps during the day, then you should be able to see screenshots.

Let me know if you have any further questions or more details.

:)

**Reply by stefanwerfling** [reply]
@jacksonm36 It's not still finished, but you can see if you understand how to proceed now: https://flying-fish.gitbook.io/flyingfish/index/configurations

I'll try to deliver the rest as soon as possible.

**Comment by stefanwerfling** [comment]
@jacksonm36 i have add your question for wildcard, https://github.com/stefanwerfling/flyingfish/issues/35

You can also have another certificate automatically issued for each subdomin you manage.

**Comment by stefanwerfling** [comment]
Now have the typo: https://github.com/stefanwerfling/flyingfish/issues/50
